plugins/dummy_rs/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "dummy_rs"
+version = "0.1.0"
+edition = "2024"
+
+[lib]
+crate-type = ["cdylib"]
+path = "src/libdummy_rs.rs"
+
+[dependencies]
+falco_plugin = { git = "https://github.com/falcosecurity/plugin-sdk-rs", tag = "v0.4.2" }
```

There are some important fixes queued up for an upcoming 0.4.3, so you might as well use the crates.io version and switch to 0.4.3 (with no Cargo.toml changes, just a `cargo update`) once it's released.

(using a git tag pins you to 0.4.2 forever, though if you prefer to use git, consider `tag = "release-v0.4"` to pick up the fixes eventually)

```suggestion
falco_plugin = "0.4"
```